src/main/java/com/post/menagment/services/PostOfficeServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.post.menagment.services;
+
+import com.post.menagment.model.PostOffice;
+import com.post.menagment.repository.PostOfficeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PostOfficeServiceImpl implements PostOfficeService{
+
+    @Autowired
```

Comment:
It's preferable to use Constructor-based dependency injection or Setter-based dependency injection, instead of Field-based dependency injection.

Rely on official spring documenatation
https://docs.spring.io/spring/docs/5.0.3.RELEASE/spring-framework-reference/core.html#beans-factory-collaborators